### PR TITLE
I18n translations for Header partial

### PR DESCRIPTION
**Pull request by jiteshjha**
Make sure these boxes are checked before your pull request is ready to be reviewed and merged. Thanks!

* [x] tests pass -- `rake test`
* [x] code is in uniquely-named feature branch, and has been rebased on top of latest master (especially if you've been asked to make additional changes)
* [x] pull request are descriptively named
* [x] if possible, multiple commits squashed if they're smaller changes
* [x] reviewed/confirmed/tested by another contributor or maintainer
* [x] `development.sqlite.example` has been updated if any database migrations were added

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/wiki/contributing-to-public-lab-software

We have a loose schedule of reviewing and pulling in changes every Tuesday and Friday, and publishing changes on Fridays. Please alert developers on plots-dev@googlegroups.com when your request is ready or if you need assistance.

Thanks!

---

@jywarren @ebarry Please look into this, thank you! :smile:

### CODE REVIEW

File: app/assets/stylesheets/style.css [hunk 1]
```diff
@@ -282,6 +283,11 @@ body { padding-top: 70px; }
   background-clip: padding-box;
 }
 
+html[lang="de"] .chat-popover {
```
Comment (jywarren) [inline app/assets/stylesheets/style.css]: Wow, I'm unfamiliar with this. Does it have wide browser support? Cool.

**Comment by jywarren** [comment]
I think this looks good. @ebarry were you able to test out the footer?

Jitesh - what's the standard way to view a different translation from the same browser?

File: app/assets/stylesheets/style.css (hunk 1 above)
Comment (jiteshjha) [inline app/assets/stylesheets/style.css]: Yes, it does. http://www.w3schools.com/tags/att_global_lang.asp

**Comment by jiteshjha** [comment]
For now, I have set up `?locale=lang` to view a different translation, just for testing purpose. Manual set up(like a dropdown option on the nav bar) and automatic set up of locale will come up in the following PR, as my next task. :smile:

**Comment by jywarren** [comment]
super!

**Comment by jywarren** [comment]
Rebasing and checking this now.

**Comment by jywarren** [comment]
Merged, thanks!

**Pull request closed without merging**